Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -2012,38 +2014,49 @@ public async Task UpdateImagesAsync(BaseItem item, bool forceUpdate = false)
                 try
                 {
                     size = _imageProcessor.GetImageDimensions(item, image);
+                    anyChange = image.Width != size.Width || image.Height != size.Height;
                     image.Width = size.Width;
                     image.Height = size.Height;
                 }
                 catch (Exception ex)
                 {
                     _logger.LogError(ex, "Cannot get image dimensions for {ImagePath}", image.Path);
                     size = default;
+                    anyChange = image.Width != size.Width || image.Height != size.Height;
                     image.Width = 0;
                     image.Height = 0;
                 }
 
                 try
                 {
-                    image.BlurHash = _imageProcessor.GetImageBlurHash(image.Path, size);
+                    var blurhash = _imageProcessor.GetImageBlurHash(image.Path, size);
+                    anyChange = anyChange || !blurhash.Equals(image.BlurHash, StringComparison.Ordinal);
+                    image.BlurHash = blurhash;
                 }
                 catch (Exception ex)
                 {
                     _logger.LogError(ex, "Cannot compute blurhash for {ImagePath}", image.Path);
+                    anyChange = anyChange || !string.Empty.Equals(image.BlurHash, StringComparison.Ordinal);
```

Comment:
This branch is hit when computing the blurhash fails.
```suggestion
```

Reply:
yes but if there was a blurhash before and it fails now, its considered a change. if there wasn't a blurhash before and it fails now, its not considered a change and does not need updating in the db

Reply:
Fair, but then I would prefer not comparing with string.Empty like that
```suggestion
                    anyChange = anyChange || !string.IsNullOrEmpty(image.BlurHash);
```